experimenter/experimenter/experiments/tests/api/v6/test_serializers.py

```diff
@@ -3,7 +3,7 @@
 
 from django.conf import settings
 from django.test import TestCase
-from mozilla_nimbus_shared import check_schema
```

Comment:
I don't know what `check_schema` in nimbus-shared does, but I'm assuming that pydantic's `model_validate` is close enough to the same thing?

Reply:
yes, check_schema runs json schema validation for the named schema, which is ~ equivalent to using the pydantic schemas.